core/src/main/java/com/linecorp/armeria/server/ServiceConfigBuilder.java

```diff
@@ -29,6 +29,28 @@
 
 final class ServiceConfigBuilder {
 
+    static ServiceConfigBuilder copyOf(ServiceConfigBuilder original) {
+        final ServiceConfigBuilder builder = new ServiceConfigBuilder(original.route, original.service)
+                .requestContentPreviewerFactory(original.requestContentPreviewerFactory)
```

Comment:
I think this can raise an NPE because the `original.requestContentPreviewerFactory` is filled when calling `VirtualHostBuilder::Build` which is later than `copyOf()`

Reply:
I think we don't need to check `null` because the original logic does not check `null`. WDYT? 
https://github.com/line/armeria/blob/13ff37ebbacaef35bcdd7b7040519ade0418bff4/core/src/main/java/com/linecorp/armeria/server/ServiceConfigBuilder.java#L118-L122

Reply:
Ah didn't notice that. Thanks!